### ISSUE TITLE
Add Hyperstack mention?

### ISSUE THREAD

**Issue by pyrareae** [vindarel/awesome-no-js-web-frameworks#2]
I noticed that hyperstack is left out. It is much more actively developed than the other Ruby frameworks mentioned and is becoming quite nice to use, and would definitely make a good addition to the list I think.

**Comment by vindarel** [comment]
Thanks, it looks incredible. I'll add it and accept a PR.

How's your experience with it? It seems you at least wrote a demo app with it? 

I tried to run it with no success tonight. I installed it following https://docs.hyperstack.org/installation/man-installation, running
 
  rails new hyperstacktest -T -m https://rawgit.com/hyperstack-org/hyperstack/edge/install/rails-webpacker.rb

then `foreman start` had no Procfile, which I created:

```
web:        bundle exec rails s -b 0.0.0.0
hot-loader: bundle exec hyperstack-hotloader -d app/hyperstack
```

but then `localhost:5000` was still unreachable. Same with BASE_URL="http://localhost:5000" foreman start

It seems that more up to date docs are here: https://hyperstack.org/edge/docs/installation/installation#createing-a-new-app-with-a-rails-template

I'm also a bit calmed by the missing features: https://docs.hyperstack.org/feature_matrix

TLDR; it's pre-alpha software to keep an eye on probably.

**Comment by pyrareae** [comment]
Yeah I took the plunge and started using it to start prototyping a project I hope to see in production eventually actually. The most recent docs are probably most reliably from the [docs directory](https://github.com/hyperstack-org/hyperstack/tree/edge/docs) in the main repository. I'm not really sure why it wasn't starting for you, but the slack support has always been extremely helpful for me when I encounter a snag like that. The initial generator seems like it's possibly still a work in progress.

This is my current procfile, which is slightly different 
```
web:        bundle exec rails s -b 0.0.0.0
hot-loader: bundle exec hyperstack-hotloader -p 25222 -d app/hyperstack
```

The missing features list has some things that seem strange to be missing, but I believe that's mostly for the automatic isomorphic api, you can also define methods that explicitly only run on the server which use the full standard rails api. The opal wrapper around react (reactrb) is imo is quite good and takes a lot of the pain out of react once you get the hang of it.

**Comment by vindarel** [comment]
thanks for the feedback. Do you know of real world and open source examples? (besides the website itself (https://github.com/hyperstack-org/website)).

**Comment by pyrareae** [comment]
[catprint](https://www.catprint.com/) I believe is built by / started by the project lead and seems like it is pretty complex from comments I've seen him make.
